products/BellHybrid/apps/application-bell-main/layouts/HomeScreenLayoutClassic.hpp

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include "layouts/BaseHomeScreenLayoutProvider.hpp"
+
+namespace style::homescreen_classic
+{
+    constexpr inline auto bottom_box_w = 390U;
+    constexpr inline auto bottom_box_h = 102U;
+} // namespace style::homescreen_classic
+namespace gui
+{
+
+    class BellBaseLayout;
+    class TextFixedSize;
+    class AlarmSetSpinner;
+    class TimeSetFmtSpinner;
+    class SnoozeTimer;
+    class BellBattery;
+    class DuoHBox;
+
+    enum class LayoutClassicVersion
+    {
+        Temperature,
+        AmPm,
+        Battery
+    };
+
+    enum class HeaderViewMode
+    {
+        Empty,
+        AlarmIcon,
+        AlarmIconAndTime,
+        SnoozeIcon,
+        SnoozeIconAndTime,
+    };
+
+    namespace battery {
+        constexpr auto chargingLevelHideBot = 100;
+        constexpr auto dischargingLevelShowTop = 20;
```

Comment:
Shouldn't the naming be Bot as well?

Reply:
no - cos we have only upper limit here - when battery level is lower than 20 we always show battery icon